### PR TITLE
edited cars new page

### PR DESCRIPTION
**Pull request by directtoanne**
Untitled

### CODE REVIEW

**Review by JakeM93 — APPROVED** [review]
Looks good to me 👍

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
   def create
     @booking = Booking.new(booking_params)
-    @booking.car = @car
-    @booking.user = @user
+    @booking.car = @car(params: car_id)
+    @booking.user = current_user
     if @booking.after_save 
-      redirect to booking_path
+      redirect to car_booking_path
```
Comment (JakeM93) [inline app/controllers/bookings_controller.rb:12]: redirect_to car_bookings_path(@car) possibly

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
   def create
     @booking = Booking.new(booking_params)
-    @booking.car = @car
-    @booking.user = @user
+    @booking.car = @car(params: car_id)
+    @booking.user = current_user
     if @booking.after_save 
-      redirect to booking_path
+      redirect to car_bookings_path(@car)
```
Comment (oceanelf) [inline app/controllers/bookings_controller.rb]: `redirect_to car_bookings_path(@car)`

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
   def new
     @booking = Booking.new
+    @car = Car.find(params[:car_id])
   end
 
   def create
     @booking = Booking.new(booking_params)
-    @booking.car = @car
-    @booking.user = @user
+    @booking.car = @car(params: car_id)
```
Comment (oceanelf) [inline app/controllers/bookings_controller.rb]: don't think we need that - we have params method

File: app/controllers/bookings_controller.rb [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
     @booking = Booking.new
+    @car = Car.find(params[:car_id])
   end
 
   def create
     @booking = Booking.new(booking_params)
-    @booking.car = @car
-    @booking.user = @user
+    @booking.user = current_user
     if @booking.after_save
```
Comment (oceanelf) [inline app/controllers/bookings_controller.rb]: `save` instead of `after_save`

**Review by oceanelf — APPROVED** [review]
love it! 😍

**Pull request merged**